ietf/group/views.py

```diff
@@ -437,6 +438,48 @@ def prepare_group_documents(request, group, clist):
 
     return docs, meta, docs_related, meta_related
 
+
+def get_leadership(group_type):
+    people = Person.objects.filter(
+        role__name__slug="chair",
```

Comment:
Not asking for a change now, but do think about the future when we might want to add IAB programs to these pages - we'll need to look for other role names at that point.